docs/users-guide/13-embedding.md

```diff
@@ -0,0 +1,62 @@
+## Embedding Metabase in other Applications
+
```

Comment:
I'd like us to add a simple overview of the feature before diving into the key concepts.

Something along the lines of:

"Metabase now includes a powerful application embedding feature that allows you to embed your saved questions or dashboards in your own web applications. You can even pass parameters to these embeds to customize them for different users."